pkg/controller/account/account_controller.go

```diff
@@ -194,8 +188,8 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 			return reconcile.Result{Requeue: true}, r.statusUpdate(currentAcctInstance)
 		}
 		// The goroutines happened in this invocation. Time out if that has taken too long.
-		if currentAcctInstance.IsOlderThan(initPendTime) {
-			errMsg := fmt.Sprintf("Initializing regions for longer than %d seconds", initPendTime)
+		if time.Now().Sub(irCond.LastTransitionTime.Time) > regionInitTime {
```

Comment:
One of the golangci-linter rules suggests using `time.Since(...)` rather than `time.Now().Sub(...)`.

```suggestion
		if time.Since(irCond.LastTransitionTime.Time) > regionInitTime {
```

Reply:
Done, thanks.